content/english/python-pixel/Activities/Activity-1.md

```diff
@@ -1,6 +1,44 @@
 ---
-title: "Create a color board"
+title: "Activity 1: Create a color board"
```

Comment:
Can you rename to activity-1.md? Not sure Github would accept the file rename though.